sdk/media/azure-media-livevideoanalytics-edge/docs/DevTips.md

```diff
@@ -0,0 +1,40 @@
+## How to update the lva sdk
```

Comment:
I see the value of this file, we need our Py architect to decide what's the best way to put it (repo is intended for customers)